main.py

```diff
@@ -0,0 +1,21 @@
+def menu():
+    """This function displays the menu and either continues to the game or quits the program."""
+    while True:
+        print("MENU \n1. New Game \n2. Exit")
+        choice = input()
+        if choice == 1:
+            print("Entering The Cell...")
+            return
+        elif choice == 2:
+            print("Thanks for playing!")
+            exit()
```

Comment:
do not call exit(), just return. 

it is better code structure if you have only 1 termination point in your code: end of main function. calling the exist system call from multiple different places can create hard to find bugs and not good for maintainability